terraform_apply.sh

```diff
@@ -0,0 +1,138 @@
+export AWS_SHARED_CREDENTIALS_FILE
+export AWS_DEFAULT_REGION
+
+# Perform a targeted apply to create the EC2 instances, then create
+# everything else.  This is a workaround for the dynamic-ish for_each
+# expressions mentioned above.
+terraform apply "${@}" \
+  -target=aws_iam_role_policy_attachment.provisionassessment_policy_attachment \
+  && terraform apply "${@}" \
+    -target=aws_default_route_table.operations \
+    -target=aws_efs_access_point.access_point \
+    -target=aws_efs_file_system.persistent_storage \
+    -target=aws_efs_mount_target.target \
+    -target=aws_iam_role_policy.gophish_assume_delegated_role_policy \
+    -target=aws_iam_role_policy.guacamole_assume_delegated_role_policy \
+    -target=aws_iam_role_policy.nessus_assume_delegated_role_policy \
+    -target=aws_iam_role_policy.teamserver_assume_delegated_role_policy \
+    -target=aws_iam_role_policy.terraformer_assume_delegated_role_policy \
+    -target=aws_iam_role_policy_attachment.efs_mount_policy_attachment_assessorportal \
+    -target=aws_iam_role_policy_attachment.efs_mount_policy_attachment_debiandesktop \
+    -target=aws_iam_role_policy_attachment.efs_mount_policy_attachment_gophish \
+    -target=aws_iam_role_policy_attachment.efs_mount_policy_attachment_kali \
+    -target=aws_iam_role_policy_attachment.efs_mount_policy_attachment_pentestportal \
+    -target=aws_iam_role_policy_attachment.efs_mount_policy_attachment_samba \
+    -target=aws_iam_role_policy_attachment.efs_mount_policy_attachment_teamserver \
+    -target=aws_iam_role_policy_attachment.efs_mount_policy_attachment_terraformer \
+    -target=aws_internet_gateway.assessment \
+    -target=aws_nat_gateway.nat_gw \
+    -target=aws_network_acl.operations \
+    -target=aws_network_acl.private \
+    -target=aws_network_acl_rule.operations_egress_to_anywhere_via_any_port \
+    -target=aws_network_acl_rule.operations_ingress_from_anywhere_via_ports_1024_thru_3388 \
+    -target=aws_network_acl_rule.operations_ingress_from_anywhere_via_ports_3390_thru_50049 \
+    -target=aws_network_acl_rule.operations_ingress_from_anywhere_via_ports_50051_thru_65535 \
+    -target=aws_network_acl_rule.private_egress_to_anywhere_via_http \
+    -target=aws_network_acl_rule.private_egress_to_anywhere_via_https \
+    -target=aws_network_acl_rule.private_egress_to_operations_via_ephemeral_ports \
+    -target=aws_network_acl_rule.private_ingress_from_anywhere_else_efs \
+    -target=aws_network_acl_rule.private_ingress_from_operations_efs \
+    -target=aws_network_acl_rule.private_ingress_from_operations_smb \
+    -target=aws_network_acl_rule.private_ingress_to_tg_attachment_via_ipa_ports \
+    -target=aws_route.cool_operations \
+    -target=aws_route.cool_private \
+    -target=aws_route.external_operations \
+    -target=aws_route.external_private \
+    -target=aws_route53_vpc_association_authorization.assessment_private \
+    -target=aws_route_table_association.private_route_table_associations \
```

Comment:
Are these sorted? I'm not a big fan of a number coming between symbols if they are.

Reply:
Please see commit d76d808.